src/pkg/cloudclient/restapi/client.go

```diff
@@ -101,5 +103,8 @@ type HttpError struct {
 }
 
 func (err *HttpError) Error() string {
-	return fmt.Sprintf("HTTP error %d (%s)", err.StatusCode, err.Message)
+	if err.Message != "" {
+		return fmt.Sprintf("%s (HTTP error %d)", err.Message, err.StatusCode)
+	}
+	return fmt.Sprintf("HTTP error %d (%s)", err.StatusCode, http.StatusText(err.StatusCode))
```

Comment:
at least let's make it consistent so it doesn't look like a bug when we don't return error messages. make it always "<message or status text> (HTTP error %d)"